index.html

```diff
@@ -412,8 +412,8 @@
 there is a risk that no alternative means of access to services are provided.
 This leads to exclusion.
 
-It should not be possible to refuse service to a person
-based on their refusal or inability to provide a digital credential.
+Websites should always provide a way to get service
+without presenting a digital credential.
```

Comment:
I think the previous sentence was better as it's more specific, in my opinion. This new wording weakens the statement, what does "provide a way to get service" mean? A company could provide an intentionally poor service and still meet this requirement, however I don't think TAG should define "poor service" but I also think we should have a stronger statement as the previous statement was.